hack/crds-to-consul-helm/main.go

```diff
@@ -77,7 +77,7 @@ func realMain(helmPathAbs string) error {
 			`    release: {{ .Release.Name }}`,
 			`    component: crd`,
 		}
-		withLabels := append(splitOnNewlines[0:9], append(labelLines, splitOnNewlines[9:]...)...)
+		withLabels := append(splitOnNewlines[0:7], append(labelLines, splitOnNewlines[7:]...)...)
```

Comment:
This ensure the labels are correctly copied into the YAML template that is created in helm